production/ksonnet/loki/config.libsonnet

```diff
@@ -12,16 +12,21 @@
 
     grpc_server_max_msg_size: 100 << 20,  // 100MB
 
-    wal_enabled: true,
     query_scheduler_enabled: false,
     overrides_exporter_enabled: false,
 
-    // flags for running ingesters/queriers as a statefulset instead of deployment type.
-    // WAL enabled configurations automatically use statefulsets.
-    stateful_ingesters: false,
+    // flags for running queriers as a statefulset instead of deployment type.
     ingester_pvc_size: '10Gi',
     ingester_pvc_class: 'fast',
 
+    ingester_data_disk_size: self.ingester_pvc_size,  // keep backwards compatibility
```

Comment:
Looks like this is the variable that is actually used:
https://github.com/grafana/loki/blob/4eddc0e08b9d0728d74766c6190b1e5326befc37/production/ksonnet/loki/ingester.libsonnet#L15
https://github.com/grafana/loki/blob/4eddc0e08b9d0728d74766c6190b1e5326befc37/production/ksonnet/loki/multi-zone.libsonnet#L68

Provided we are deprecating stuff, should we remove `ingester_data_disk_size` and only have `ingester_pvc_size` to keep things simple?

Reply:
Since `ingester_data_disk_size` is referenced, why do you want to replace it with the other variable?

Reply:
I was proposing to remove ingester_data_disk_size and only use ingester_pvc_size.